### ISSUE TITLE
Inventory Changer and

### ISSUE THREAD

**Issue by mliiyyyyy** [danielkrupinski/Osiris#3804]

**Comment by Fnoberz** [comment]
Have you installed all Runtime versions?

**Comment by mliiyyyyy** [comment]
> Have you installed all Runtime versions?

no, i mean 'enable shuffle to equit ramdom item each map in csgo' is not work with Inventory Changer.
of curse, my all skin is made by Inventory Changer.
